webhook-bridge/svix-webhook-bridge.example3.yaml

```diff
@@ -0,0 +1,74 @@
+log_level: "debug"
+log_format: "json"
+
+# Optionally configure these settings to export OTEL data
+opentelemetry:
+  address: "http://localhost:1234"
+  service_name: "my-bridge"
+  sample_ratio: 0.5
+
+# Host/port to listen on when receiving webhooks
+receiver_http_addr: "0.0.0.0:5000"
+
+# Global svix settings.
+# A svix client is needed by each sender.
+# Each sender can override any of these fields by defining a `svix_client` section.
+global_svix_client:
```

Comment:
I'm on the fence about having a global here. I can definitely see the value but also maybe adds complexity given that most people will probably just have one receiver/sender so it doesn't add much.

Reply:
Can definitely leave it out for now. Just wanted to float the idea.